ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -553,7 +553,9 @@ public void setTextShadowColor(int textShadowColor) {
 
   @ReactProp(name = PROP_TEXT_TRANSFORM)
   public void setTextTransform(@Nullable String textTransform) {
-    if (textTransform == null || "none".equals(textTransform)) {
+    if (textTransform == null) {
+      mTextTransform = TextTransform.UNSET;
+    } else if("none".equals(textTransform)) {
```

Comment:
Nit: Missing a space between the `else if` and the open parenthesis.

```suggestion
    } else if ("none".equals(textTransform)) {
```